products/pact-act-wizard/engineering/testing-qa.md

```diff
@@ -0,0 +1,84 @@
+# QA Artifact for PACT Act app
+MVP launching TBD
+
+## Overview of app and QA risks
+The PACT Act app has moderate complexity from a testing perspective:
+- Not authenticated
+- Uses design system components
+- No APIs
+- Branching logic for display of questions and results pages
+
+### React application complexity: moderate
+The first question in the application is "When did you serve in the U.S. military?"
+
+Based on the response, the user will be asked between 1 and 9 questions. We refer to this as "branching logic." There are two different answer types:
+
+- Multi-select checkboxes (1 or more response)
+- Radio buttons (1 response)
+
+Every question also has its own branching logic. For instance, if a user selects "Yes" to a question, they will be guided down a different question path
+or to a results screen. A different path exists for a "No" or "I'm not sure" response.
+
+[This MURAL](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692989444688/0044b9825c82d8d23920601f68c41a61d047d681?sender=ue51e6049230e03c1248b5078) illustrates how the questions are connected to one another and to the results screens at the end.
+
+## Our QA approach
+**UX / interaction**
+
+During development the team will manually test every user story that is implemented. So for example, the branching logic (the methods by which the question display is determined) will be thoroughly tested through a few different paths based on question responses.
+
+We will also add automated tests to cover these cases to prevent regression. 
+
+## Manual test cases
+**UX validation**
+
+- Navigation through most possible flows (all including the appropriate results screen)
+  - "Yes" or checkbox(es) selected for each question
+  - "No" or "I'm not sure" selected for each question
+  - Combinations of the above
+  - Results screen accordions are validated for proper open/close behavior as well as content
+- Validation errors as expected
+  - Yes/No/I'm not sure questions - one response must be selected to continue
+  - Multi-checkbox questions - at least one response must be selected to continue`*`
+- Deep linking prevention
+  - If a user lands within the flow for any reason, they are redirected to the /introduction screen.
+
+`*` For multi-checkbox questions that have a "None of these locations" or "I'm not sure" response, we will accept only:<br/>
```

Comment:
Randi, there is late-breaking news on these questions – we will no longer have weird checkbox logic like this. All questions with a No or Not Sure response option will be radio buttons! so you can delete this bit

Reply:
Thanks @wesrowe!